releasenotes/notes/submodule-update-a2f6c5e9cb597a99.yaml

```diff
@@ -0,0 +1,16 @@
+---
+critical:
+  - |
+    In the newton and newton-rc branches openstack-ansible is referenced as a
+    submodule inside of the rpc-openstack path.  When a deployer is not using a
+    fresh clone of the rpc-openstack repository it is important to ensure that
+    the openstack-ansible SHA is pointing at the latest commit.  This can be
+    accomplished with the following commands run from the root of your
+    rpc-openstack path (i.e. /opt/rpc-openstack)::
```

Comment:
Use a full stop here, not ``::``. The leave a line break, then put ``.. code-block:: shell`` and another line break, then the commands to run (with a leading two spaces to align with the ``..`` in the code-block directive.

Reply:
Done